.github/workflows/ci-base.yml

```
@@ -0,0 +1,46 @@
+name: ci-base
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - packages/base/**
+      - ./.github/workflows/ci-base.yml
+  pull_request:
+    branches: [ master ]
+    paths:
+      - packages/base/**
+      - ./.github/workflows/ci-base.yml
+jobs:
+  ci-base:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    defaults:
+      run:
+        working-directory: "./packages/base"
+    env:
+      NPM_CI: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Clean
+        run: npm run clean
+      - name: Install node modules
```

[IMO, aka opinionated suggestion]
```suggestion
          node-version: 16
      - name: Clean
        run: npm run clean
      - name: Install
```